customization_config/en/traits/general.yaml

```diff
@@ -0,0 +1,35 @@
+traits:
+- id: prog_lang
+  label: Programming Language
+  internal_notes: Do not use for frameworks like Next or React Native. Instead, use `env`.
+
+- id: protocol
+  label: Protocol
+  internal_notes: For example, HTTP, gRPC, or SOCKS.
+
+- id: platform
+  label: "Platform"
+  internal_notes: The target platform (language, framework, specific env) for a library, SDK, or other third-party entity. Do not use if every option is a pure programming language (Go, Python, etc.) -- use `prog_lang` instead. Re-label to "Framework", "SDK", etc. if it's more appropriate for your context.
+
+- id: host
+  label: "Host"
+  internal_notes: Hosting platform, such as AWS or GCP. Can also refer to containers like Kubernetes.
+
+- id: host_type
+  label: "Host type"
+  internal_notes: The machine or container type within a given host, such as EC2. Use only as a followup to a `host` selection.
+
+- id: os
+  label: "OS"
+  internal_notes: Env operating system, such as Linux or Windows. Can refer to a mobile OS, but `mobile_os` is preferred for lists of mobile-only options.
+
+- id: database
+  label: "Database"
+
+- id: mobile_os
+  label: "OS"
+  internal_notes: Mobile operating system, such as Android.
+  
+- id: lib_src
```

Comment:
The only change is that I renamed this from `pkg_mgr` to allow for a broader range of logical values.